parity-clib/examples/cpp/CMakeLists.txt

```diff
@@ -7,16 +7,19 @@ set (CMAKE_CXX_EXTENSIONS OFF) # Enforce C++17
 set (CMAKE_EXPORT_COMPILE_COMMANDS ON) # generate a compilation database
 add_executable(parity-example main.cpp)
 add_compile_options("-Wall -Wextra -Werror")
-
+set(parity_shared_library_path "${CMAKE_SOURCE_DIR}/../../../target/release/${CMAKE_SHARED_LIBRARY_PREFIX}parity${CMAKE_SHARED_LIBRARY_SUFFIX}")
 ExternalProject_Add(
-    libparity
+    libparity.so
```

Comment:
I don't think that's cross-platform!

Reply:
It doesn’t actually matter ― it is just an opaque string to CMake.  That said, I can revert it.